src/cli/utils.ts

```diff
@@ -341,3 +345,36 @@ export function getTagsFromOptions(
 ): { name: string; value: string }[] {
   return parseTags(options.tags);
 }
+
+export function currencyFromOptions<
+  O extends GlobalOptions & { currency?: string },
+>(options: O): Currency | undefined {
+  const currency = options.currency?.toLowerCase();
+
+  if (currency !== undefined && !isCurrency(currency)) {
+    throw new Error(
+      `Invalid fiat currency type ${currency}!\nPlease use one of these:\n${JSON.stringify(
+        fiatCurrencyTypes,
+        null,
+        2,
+      )}`,
+    );
+  }
+
+  return currency;
+}
+
+export function requiredByteCountFromOptions({
+  byteCount,
+}: TokenPriceOptions): number {
+  const byteCountValue = byteCount !== undefined ? +byteCount : undefined;
+  if (
+    byteCountValue === undefined ||
+    byteCountValue <= 0 ||
+    isNaN(byteCountValue) ||
```

Comment:
Might want this condition before attempting numeric comparison above.

Reply:
24e2d90